clients/input_copy.go

```diff
@@ -197,12 +218,25 @@ func CopyFile(ctx context.Context, sourceURL, destOSBaseURL, filename, requestID
 		byteAccWriter := ByteAccumulatorWriter{count: 0}
 		defer func() { writtenBytes = byteAccWriter.count }()
 
+		var c io.ReadCloser
 		c, err := getFile(ctx, requestID, sourceURL, dStorage)
+
 		if err != nil {
-			return fmt.Errorf("download error: %w", err)
+			glog.Errorf("error getting file: %w", err)
+			return err
```

Comment:
I think the old behavior of wrapping the error with a more specific error is cleaner. It will be logged on the outer layer
```suggestion
			return fmt.Errorf("download error: %w", err)
```